cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -413,18 +413,75 @@ func metricsListGrabAggregateConvert(ctx context.Context, t metricsTarget) (map[
 	if len(targetList) > 1 {
 		resetDetected = false
 	}
-
+	
 	return families, interestingMetrics, resetDetected, nil
 }
 
 // Handler grabs metrics from target, aggregates the metrics and convert them into cloudwatch metrics
 func Handler(ctx context.Context, t metricsTarget) {
 	families, interestingMetrics, resetDetected, err := metricsListGrabAggregateConvert(ctx, t)
 
+	updatePromMetrics(interestingMetrics)
+
 	if err != nil || resetDetected {
 		t.getLogger().Infof("Skipping 1st poll after reset, error: %v", err)
 	}
 
 	cw := t.getCWMetricsPublisher()
 	produceCloudWatchMetrics(t, families, interestingMetrics, cw)
 }
+
+func updatePromMetrics(interestingMetrics map[string]metricsConvert){
```

Comment:
Instead of manually calling metrics, we should loop over all metrics similar to how `produceCloudWatchMetrics` function does. Maybe we can move both into the same function and use env var to control updates.

Reply:
This will also simplify the process of adding new metrics.